framework/test/randoop_coverage.sh

```diff
@@ -22,6 +22,7 @@ if [ ! -f test.include ]; then
 fi
 source test.include
 init
+export TMP_DIR
```

Comment:
This seems unnecessary. TMP_DIR isn't used elsewhere, is it?

Reply:
It would be better to be able to run `randoop_coverage.sh` and then run a separate script.  `show_coverage.pl` is just one example of such a script.  But currently there is no way to know where `randoop_coverage.sh` left its results.
It would be better for it to either take an argument indicating what directory to use, or to communicate the directory to clients.

Reply:
Can you parameterize the script(s) so that they take the temporary working directory as an argument?